hphp/runtime/ext/fb/ext_fb.php

```diff
@@ -91,7 +91,7 @@ function fb_rename_function(string $orig_func_name,
  * @param mixed $input - What string to sanitize.
  * @return bool - Sanitized string.
  */
-<<__HipHopSpecific, __Native>>
+<<__HipHopSpecific, __Native, __IsFoldable>>
 function fb_utf8ize(mixed &$input): bool;
```

Comment:
I don't see where you're dealing with reference parameters... won't this just "optimize" it to the bool result, while failing to sanitize $input?